frontend/src/common/hooks/useCategoryFilter.ts

```diff
@@ -821,14 +821,14 @@ function buildRangeCategoryView(
   rangeCategory: RangeCategory
 ): RangeCategoryView {
   // Build view model of range category.
-  const rangeView = {
+  const rangeView: RangeCategoryView = {
     key: categoryKey as CategoryKey,
```

Comment:
Don't we already have `categoryKey` typed as a result of the arg typing?

Reply:
Good catch! I'll update here as well as the corresponding areas for select and ontology category views.